assign1/src/App.css

```diff
@@ -0,0 +1,52 @@
+.App {
```

Comment:
while naming CSS classes, follow lower case pattern i.e .app, .app-logo

Reply:
actually this folder is automatically created with the app, and the css written by me is in index.css file.

Reply:
this css class name are auto generated while created the app, and i got the point, i will change it lowercase .